ups/eupspkg.cfg.sh

```diff
@@ -1,3 +1,18 @@
 # EupsPkg config file. Sourced by 'eupspkg'
 
-PYSETUP_INSTALL_OPTIONS="--prefix $PREFIX --standalone-lib"
+build()
+{
+    python2.7 setup.py build
+}
+
+install()
+{
+    clean_old_install
+    python2.7 setup.py install --prefix "$PREFIX" --standalone-lib
+
+    # Install ups if the native build system hasn't done it already.
+    # We do this check to avoid expanding the table file twice (EUPS has a bug there)
```

Comment:
This code was cargo-culted from EUPS itself by looking to see what the default `install()` function did so I could make sure I did the same in my version. See RobertLuptonTheGood/eups@9859a7fe65e639b947254ad55473ddd66b462829 -- I'll change the comment to just say where the code came from. I see that the 7 other packages in the stack with their own install overrides do not do this check (but that may or may not be right).